src/main/java/com/example/bookstore/model/Author.java

```diff
@@ -0,0 +1,4 @@
+package com.example.bookstore.model;
+
+public class Author {
```

Comment:
can you add private fields as well in this PR?

Reply:
I need to add private fields in the PR, without changing anything else in the code. Is that correct?

Reply:
Yeah, exactly. You need to add attributes to this class. i.e. Name, birthdate, country, language and others.

Reply:
I dont understud all, can You check now. And I dont have in code country, to add it?